app/src/main/kotlin/com/wire/android/ui/home/conversations/details/editguestaccess/EditGuestAccessViewModel.kt

```diff
@@ -174,7 +174,11 @@ class EditGuestAccessViewModel @Inject constructor(
                         )
                     )
 
-                    is UpdateConversationAccessRoleUseCase.Result.Success -> Unit
+                    is UpdateConversationAccessRoleUseCase.Result.Success -> updateState(
```

Comment:
This test was failing, because we were not explicitly updating the State in the success case. 
Test caught this unexpected behavior in a TDD-like fashion